tests/io/v3/connectors/test_connectors_api.py

```diff
@@ -0,0 +1,584 @@
+            'params': {
+                'sub_accounts': [
+                    {
+                        'account_id': '012345678910',
+                        'trails': [],
+                        'role_arn':
+                            'arn:aws:iam::00:role/tenableio-connector_example',
+                        'external_id': '7930d8cc-27c0-43cc-abdb-b846e2b13414',
+                        'incremental_mode': False
+                    }
+                ],
+                'import_config': False,
+                'auto_discovery': True,
+                'tags': [
+                    {
+                        'key': 'Tenable',
+                        'value': 'FA'
+                    }
+                ],
+                'service': 'aws_keyless'
+            },
+            'network_id': '00000000-0000-0000-0000-000000000000',
+            'last_seen_updated': '2021-07-22T14:31:39.801Z'
+        }
+    }
+
+    connector_id = 'b3397cb2-6650-432b-ae1c-fc64ec7ef6ba'
+    responses.add(
+        responses.GET,
+        f'{CONNECTORS_BASE_URL}/{connector_id}',
+        json=resp_data
+    )
+
+    resp = api.v3.connectors.details(connector_id)
+    assert resp['id'] == connector_id
+
+
+@responses.activate
+def test_download_template(api):
+    '''
+    Test the download template function
+    '''
+
+    resp_data = {
+        '$schema':
+            'https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json#', # noqa E501
+        'contentVersion': '1.0.0.0',
+        'variables': {
+            'connectorId': 'b3397cb2-6650-432b-ae1c-fc64ec7ef6ba',
+            'storageAccount': 'faconnector',
+            'storageContainer': 'ba10492c-365b-1a45-ba07-5670efae1b12',
+            'storageKey': 'redacted',
+            'scriptsAccount': 'faautomationscript',
+            'scriptsKey': 'redacted',
+            'tableKey': 'redacted',
+            'keyRotationMethod': 'default',
+            'resourceGroupLocation': 'eastus',
+            'resourceGroupName':
+                '[concat("Tenable-FA-Connector-",variables("connectorid"))]',
+            'tenableFACollectionRoleDefinitionResourceName':
+                '[guid("Tenable-FA-Custom-Role-Def",'
+                ' subscription().id, variables("connectorId"))]',
+            'tenableFACollectionRoleDefinitionId':
+                '[subscriptionResourceId('
+                '"Microsoft.Authorization/roleDefinitions",'
+                ' variables("tenableFACollectionRoleDefinitionResourceName")'
+                ')]',
+            'logicAppName': 'TenableFAEventBasedCollection',
+            'resourceTags': {
+                'Tenable': 'AzureFa'
+            },
+            'roleDefinition': {
+                'name':
+                    '[format("Tenable FA Role (Subscription: {0} |'
+                    ' Connector: {1})", subscription().subscriptionId, '
+                    'variables("connectorId"))]',
```

Comment:
Please replace " -> ' at all instances

Reply:
I guess we are using single quotes at all place.